src/DocumentFormat.OpenXml.Framework/OpenXmlPartWriter.cs

```diff
@@ -365,5 +471,179 @@ public override void Close()
         }
 
         #endregion
+
+        // Async Methods
+#if FEATURE_ASYNC_SAX_XML
+        /// <summary>
+        /// Asynchronously writes the XML declaration with the version "1.0".
+        /// </summary>
+        public async override Task WriteStartDocumentAsync()
+        {
+            ThrowIfObjectDisposed();
+            await _xmlWriter.WriteStartDocumentAsync().ConfigureAwait(true);
```

Comment:
do these async methods take a CancellationToken? If so, we should be taking that in as well.

Reply:
I double-checked, they don't take a CancellationToken